src/ext/php5_4/dispatch.c

```diff
@@ -58,30 +58,3 @@ zend_bool ddtrace_dispatch_store(HashTable *lookup, ddtrace_dispatch_t *dispatch
     return zend_hash_update(lookup, Z_STRVAL(dispatch->function_name), Z_STRLEN(dispatch->function_name), &dispatch,
                             sizeof(ddtrace_dispatch_t *), NULL) == SUCCESS;
 }
-
```

Comment:
This caused an error due to `-Werror`. It seems not to be used and it is safe to be removed.

Reply:
Nice! I thought I already removed it, but turns out this is just a similar function in a similar place. Good riddance :)